include/xten/Conversion/CMakeLists.txt

```diff
@@ -9,12 +9,12 @@ set(LLVM_TARGET_DEFINITIONS Passes.td)
 mlir_tablegen(Passes.h.inc -gen-pass-decls -name XTenConversion)
 add_public_tablegen_target(XTenConversionIncGen)
 
-set(LLVM_TARGET_DEFINITIONS ATenToXTen.td)
-mlir_tablegen(ATenToXTen.cpp.inc -gen-rewriters)
-add_public_tablegen_target(ATenToXTenIncGen)
+# set(LLVM_TARGET_DEFINITIONS ATenToXTen.td)
```

Comment:
Please add a comment here to explain why the lines are commented out (maybe refer to comment in `include/xten/Conversion/ATenToXTen.td`).